serverless/functions/serverless-functions-reference-guide.adoc

```diff
@@ -15,7 +15,8 @@ include::modules/technology-preview.adoc[leveloffset=+2]
 * xref:../../serverless/functions/serverless-developing-nodejs-functions.adoc#serverless-developing-nodejs-functions[Node.js]
 * xref:../../serverless/functions/serverless-developing-python-functions.adoc#serverless-developing-python-functions[Python]
 * xref:../../serverless/functions/serverless-developing-go-functions.adoc#serverless-developing-go-functions[Golang]
-* Quarkus
+* xref:../../serverless/functions/serverless-developing-quarkus-functions.adoc#serverless-developing-quarkus-functions[Quarkus]
+* xref:../../serverless/functions/serverless-developing-typescript-functions.adoc#serverless-developing-typescript-functions[TypeScript]
```

Comment:
@abrennan89 I think also this list of supported runtimes should be upgraded: 
https://github.com/abrennan89/openshift-docs/blob/SRVOCF-311/serverless/functions/serverless-functions-about.adoc

Reply:
Thanks @mvinkler !
cc @rh-max - FYI, when new runtimes are added, please make sure that you are adding links for them to the reference guide and "about" docs